test/test-pages/medium-1/expected.html

```diff
@@ -1,5 +1,6 @@
 <div id="readability-page-1" class="page">
     <div class="section-inner layoutSingleColumn">
+        <h2 name="3c62" id="3c62" data-align="center" class="graf--h2">Open Journalism Project:</h2>
         <h4 name="425a" id="425a" data-align="center" class="graf--h4"><em class="markup--em markup--h4-em">Better Student Journalism</em></h4>
```

Comment:
Funny, here we should really be taking out the `h4` *and* the `h2`. Anyway, we're not making things worse, so...